### ISSUE TITLE
sum 안에서 epsilon 더하는 것과 sqrt에서 epsilon 더하는 거 둘다 차이는 크지 않을 것 같긴함

### ISSUE THREAD

**Issue by ShinYwings** [yukyeongmin/model#3]
https://github.com/yukyeongmin/model/blob/c1584ba5f5604f07ddc2964ee2bce5810f7f86f7/pix2pix/pix2pix_train.py#L46

# Reference
https://github.com/EmilienDupont/wgan-gp/blob/ef82364f2a2ec452a52fbf4a739f95039ae76fe3/training.py#L73
> gradients_norm = torch.sqrt(torch.sum(gradients ** 2, dim=1) + 1e-12)
^^7

**Comment by yukyeongmin** [comment]
별차이 없겠지만 그래도 수용합니다.
